index.js

```diff
@@ -13,6 +13,9 @@ export default function toCSS(obj) {
       value = `${value}px`;
     }
 
-    return `${key}:${value} !important;`;
+    const isTouch = ('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch;
```

Comment:
let's go for this now and get Vibhor to test it on the potentially problematic devices and see what happens, shall we?